lib/puppet/type/rds_db_option_group.rb

```diff
@@ -0,0 +1,29 @@
+Puppet::Type.newtype(:rds_db_option_group) do
+  @doc = 'Type representing an RDS DB Option group.'
+
+  newparam(:name, namevar: true) do
+    desc 'The name of the DB Option Group (also known as the db_option_group_name).'
+    validate do |value|
+      fail 'name should be a String' unless value.is_a?(String)
+    end
+  end
+
+  newproperty(:description) do
+    desc 'The description of a DB option group.'
+    validate do |value|
+      fail 'description should be a String' unless value.is_a?(String)
+    end
+  end
+
+  newproperty(:options) do
+    desc 'The options'
```

Comment:
Yes, @zleswomp After revisiting my changes, I've realized that I never finished the "options" enumeration in rds_db_option_group. So let's no merge just yet as I do some more . That being said, the feature added to rds_instance looks to be working great.